src/main/java/de/retest/recheck/persistence/SeparatePathsProjectLayout.java

```diff
@@ -0,0 +1,46 @@
+package de.retest.recheck.persistence;
+
+import static de.retest.recheck.Properties.GOLDEN_MASTER_FILE_EXTENSION;
+import static de.retest.recheck.Properties.TEST_REPORT_FILE_EXTENSION;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Uses two separate paths (provided in the constructor) to place the Golden Masters in a (usually git-managed) resource
+ * folder, but the reports in a temporary build-artefacts folder.
+ *
+ * The naming of the Golden Masters follows <code>${TEST_CLASS_NAME}/${TEST_METHOD_NAME}.${STEP_NAME}</code>, whereas
+ * test report files only use <code>${TEST_CLASS_NAME}</code>.
+ */
+public class SeparatePathsProjectLayout implements ProjectLayout {
```

Comment:
```suggestion
public class WithinBaseFolderProjectLayout implements ProjectLayout {
```

Reply:
Hmm. I would argue that he main feature of this class is to have the GoldenMaster and the reports in two distinct folders. Hence the name...